### PR TITLE
User sign in

### PR DESCRIPTION
**Pull request by blafri**
Hey Felix
I Finished the user sign in assignment.
Let me know if everything looks ok and if I can merge it.

Regards,
Blayne

### CODE REVIEW

File: spec/features/user_sign_in_spec.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      fill_in 'user_password', with: 'njnjkdksnk'
+      click_button 'Log In'
+      
+      expect(page).to have_content('Invalid email or password')
+      expect(page).to have_content('Create Account')
+    end
+  end
+end
+
+feature "Sign out" do
```
Comment (felixclack) [inline spec/features/user_sign_in_spec.rb]: I would put this into it's own file, eg. `spec/features/sign_out_spec.rb`

File: spec/features/user_sign_in_spec.rb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    fill_in 'user_password', with: user.password
+    click_button 'Log In'
+    click_on 'Log Out'
+    
+    expect(page).to have_content("Signed out successfully")
+    expect(page).to have_content("Create Account")
+  end
+end
+
+feature "Reset Password" do
```
Comment (felixclack) [inline spec/features/user_sign_in_spec.rb]: This should be in it's own file too, `spec/features/reset_password_spec.rb`

**Comment by felixclack** [comment]
The code looks good.

**Comment by blafri** [comment]
Hey Felix,
I went ahead and moved the sign out and reset password feature specs to their own files.
Thanks for help. I will merge the pull request.

Thanks,
Blayne

**Pull request merged**